docs/modules/ROOT/pages/components.adoc

```diff
@@ -146,6 +146,53 @@ See {erc20-component} as an example which includes:
 - *Internal implementations*
 ====
 
+=== Mixins
+
+While separating components into granular implementations offers flexibility, integrating components with many implementations can appear crowded especially when the contract will use all of the component's implementations.
```

Comment:
what about showing two small blocks comparing with/without mixin? just the component + impls, not all the contract so it's not too large, but it shows the benefit